app/views/unattended/provisioning_templates/snippet/pxegrub2_chainload.erb

```diff
@@ -43,7 +45,9 @@ echo "the grub2_chainload template."
 echo
 echo "(*) grub2-efi-x64-2.02-122.el8 (upstream doesn't have the patches yet)"
 echo
-#connectefi scsi
+<% if host_param_true?('grub2-connectefi-scsi', true) -%>
```

Comment:
It will not work for default template, where `@host` is not defined. Maybe something like this:

```suggestion
<% if !@host || @host && host_param_true?('grub2-connectefi-scsi', true) || !@host -%>
```